packages/classes/src/__test__/Payload.unit.test.js

```diff
@@ -35,17 +35,31 @@ describe('Payload', () => {
 			expect(payload.duration).toBe(1000)
 			expect(payload.position).toBe(60000)
 		})
+		it('should populate a default createdAt', () => {
+			const payload = new Payload()
+			expect(payload.createdAt).not.toBe('')
+		})
+		it('should set createdAt differently for payloads created at different times', (done) => {
+			const firstPayload = new Payload()
+			setTimeout(() => {
+				const secondPayload = new Payload()
+				expect(firstPayload.createdAt).not.toEqual(secondPayload.createdAt)
+				done()
+			}, 10)
+		})
 	})
 
 	describe('serialize', () => {
 		it('should properly should serialize data with an empty object', () => {
 			const payload = new Payload()
+			payload.createdAt = '' // Default createdAt is dynamic so we can't snapshot the result.
```

Comment:
Consider mocking the date instead. [Here's](https://github.com/tvkitchen/tv-kitchen/blob/1efb9f6caba1efbae63483abdd1aa15e434ba96a/src/tests/ingestion.int.test.js#L17) how I did it elsewhere in the project.

Reply:
Thank you -- excellent suggestion, and hopefully this can allow me to avoid the timeout in an earlier test which would be a relief.

Reply:
@chriszs you might be interested in something I found [on StackOverflow](https://stackoverflow.com/a/47781245/159522) which is a slightly more concise version of what you did!

`jest.spyOn(Date, 'now').mockImplementation(() => 1487076708000);`